### PR TITLE
LT-2629: Migrate to new ref data

### PR DESCRIPTION
**Pull request by gponomarev-lykke**

### CODE REVIEW

**Review by tarurar — DISMISSED** [review]

File: src/MarginTrading.CommissionService.Services/Caches/ClientProfileSettingsCache.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                _lockSlim.ExitWriteLock();
+            }
+        }
+
+        public void AddOrUpdate(ClientProfileSettingsCacheModel clientProfileSettings)
+        {
+            _lockSlim.EnterWriteLock();
+            try
+            {
+                _cache[clientProfileSettings.ClientProfileId] = clientProfileSettings;
```
[the remark this answers is not in the document]
Comment (gponomarev-lykke) [reply src/MarginTrading.CommissionService.Services/Caches/ClientProfileSettingsCache.cs:66]: Yes, most likely. Will fix

File: src/MarginTrading.CommissionService/Modules/CqrsModule.cs [hunk 2]
```diff
@@ -263,6 +271,50 @@ private IRegistration RegisterAccountListenerSaga()
                     typeof(AssetPairsProjection), _settings.ContextNames.SettingsService);
         }
 
+        private void RegisterProductProjection(
+            ProcessingOptionsDescriptor<IBoundedContextRegistration> contextRegistration)
+        {
+            contextRegistration.ListeningEvents(
+                    typeof(ProductChangedEvent))
+                .From(_settings.ContextNames.SettingsService)
+                .On(EventsRoute)
```
[the remark this answers is not in the document]
Comment (gponomarev-lykke) [reply src/MarginTrading.CommissionService/Modules/CqrsModule.cs:280]: Fixed

File: src/MarginTrading.CommissionService/Modules/RabbitMqModule.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            _settings = settings;
+            _log = log;
+        }
+        
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.Register(x => new UnderlyingChangedSubscriber(x.Resolve<UnderlyingChangedHandler>(),
+                    _settings.UnderlyingChangedRabbitSubscriptionSettings.AppendToQueueName($"{_settings.InstanceId}:{_settings.BrokerId}"), _log))
+                .AsImplementedInterfaces()
```
[the remark this answers is not in the document]
Comment (gponomarev-lykke) [reply src/MarginTrading.CommissionService/Modules/RabbitMqModule.cs:28]: Fixed

File: src/MarginTrading.CommissionService/Projections/ClientProfileProjection.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+        public ClientProfileProjection(IClientProfileCache clientProfileCache,
+            IConvertService convertService)
+        {
+            _clientProfileCache = clientProfileCache;
+            _convertService = convertService;
+        }
+
+        [UsedImplicitly]
+        public async Task Handle(ClientProfileChangedEvent @event)
```
[the remark this answers is not in the document]
Comment (gponomarev-lykke) [reply src/MarginTrading.CommissionService/Projections/ClientProfileProjection.cs:28]: Why? I don't think ClientProfileContract contains any fields related to these fees. And when related ClientProfileSettings is changed, we update the fee cache.

**Review by andrey-anokhin — APPROVED** [review]

**Pull request merged**